L118-core-remove-cronet.md

```diff
@@ -0,0 +1,34 @@
+Title
+----
+* Author(s): ctiller
+* Approver: wenbozhu
+* Status: Draft
+* Implemented in: C++, Objective-C
+* Last updated: [Date]
+* Discussion at: <google group thread> (filled after thread exists)
+
+## Abstract
+
+Remove the Cronet transport from gRPC core (and consequently the Objective-C bindings to it).
+
+## Background
+
+gRPC C++, Objective-C can use Cronet on iOS as an alternative HTTP/2 stack.
+The Cronet library itself has been deprecated since September 2023, and so instead of maintaining this code we're opting to remove it.
```

Comment:
"The Cronet library itself has been deprecated _on iOS_ since..."

Reply:
updated